### ISSUE TITLE
Error installing version 1.2.0

### ISSUE THREAD

**Issue by everflux** [darklynx/request-baskets#75]
Installing version v1.2.0 results in the following error, version v1.1.0 works as expected

```
$ go install github.com/darklynx/request-baskets@latest
go: downloading github.com/darklynx/request-baskets v1.2.0
go: github.com/darklynx/request-baskets@latest: github.com/darklynx/request-baskets@v1.2.0: parsing go.mod:
	module declares its path as: darklynx/request-baskets
	        but was required as: github.com/darklynx/request-baskets
```

**Comment by darklynx** [comment]
Hi @everflux ,

Thank you for reporting an issue!

I think you are right, it seems to be an error in [go.mod](https://github.com/darklynx/request-baskets/blob/master/go.mod#L1) declaration.

I believe it should be 🤔 :
```
module github.com/darklynx/request-baskets

```

**Comment by darklynx** [comment]
it should be fixed now, please check
I get the following output:
```
# go install github.com/darklynx/request-baskets@latest
go: downloading github.com/darklynx/request-baskets v1.2.1
go: downloading github.com/go-sql-driver/mysql v1.6.0
go: downloading github.com/julienschmidt/httprouter v1.3.0
go: downloading github.com/lib/pq v1.10.7
go: downloading go.etcd.io/bbolt v1.3.6
go: downloading golang.org/x/sys v0.0.0-20220919091848-fb04ddd9f9c8

```

**Comment by everflux** [comment]
Yes, the issue is indeed solved with 1.2.1. Thank you!
